source/guide_nextcloud.rst

```diff
@@ -249,7 +249,7 @@ Add the following cronjob to your :manual:`crontab <daemons-cron>`:
 
 ::
 
- */5  *  *  *  * sleep $(( 1 + $RANDOM \% 60 )) ; php -f $HOME/html/cron.php > $HOME/logs/nextcloud-cron.log 2>&1
+ */5  *  *  *  * sleep $(( 1 + RANDOM % 60 )) ; php -f $HOME/html/cron.php > $HOME/logs/nextcloud-cron.log 2>&1
```

Comment:
The `%` needs to be escaped. From crontab's man page:

> The "sixth" field (the rest of the line) specifies the command to be run.  The entire command portion of the line, up to a new‐line  or  a  "%" character, will be executed by /bin/sh or by the shell specified in the SHELL variable of the cronfile.  A "%" character in the command, unless escaped with a backslash (\), will be changed into newline characters, and all data after  the first % will be sent to the command as standard input.

Reply:
#1479